### ISSUE TITLE
sf_4.127e-5.pkl and test results

### ISSUE THREAD

**Issue by fffpuzzle** [hacylu/Hagnifinder#3]
Hello.
I am interested in your research and tried to estimate the magnification using the BreakHis dataset and the publicly available sf_4.127e-5.pkl data, but many of the estimation results are around x40, regardless of the actual image magnification.
What dataset was sf_4.127e-5.pkl trained on?
Also, on which dataset did you train for the magnification estimation of BreakHis in your paper?

**Comment by ZHT150798** [comment]
Thank you for asking, and we have also identified this issue. Firstly, sf_4.127e-5.pkl was trained using the Hagni40 dataset. Secondly, in our paper, we fine-tuned the sf_4.127e-5.pkl model using the training set of BreakHis and subsequently tested it on the test set.
We found that the model had poor generalization ability, so we released Version 1.1. In this version, the model was trained using the hagni40_supply dataset, which includes two additional types of tumors and demonstrates better predictive performance on unseen data.
